server/sonar-server/src/main/java/org/sonar/server/organization/ws/DeleteAction.java

```diff
@@ -138,6 +137,11 @@ private void deleteQualityProfiles(DbSession dbSession, OrganizationDto organiza
     qProfileFactory.delete(dbSession, profiles);
   }
 
+  private void deleteQualityGates(DbSession dbSession, OrganizationDto organization) {
+    Collection<QualityGateDto> qualityGates = dbClient.qualityGateDao().selectAll(dbSession, organization);
```

Comment:
Here we should remove the built-in quality gate isn't it ?

Reply:
Check is already done before

Reply:
Oups, sorry, indeed no check was done, so we need to filter out the built-in QG